bigbluebutton-html5/private/config/settings.yml

```diff
@@ -639,6 +639,16 @@ public:
       throttleInterval: 400
       flushOnClose: true
       logTag: ''
+  virtualBackgrounds:
+    enabled: false
+    storedOnBBB: true
+    showThumbnails: true
+    imagesPath: /resources/images/virtual-backgrounds/
+    thumbnailsPath: /resources/images/virtual-backgrounds/thumbnails/
+    fileNames:
+      - home.jpg
+      - coffeeshop.jpg
+      - board.jpg
```

Comment:
Does it make sense to also include `blur.jpg` here so people could unlist it if undesired?

Reply:
Yes. I don't like the fact that it's hardcoded right now as well.
But it might not be that straightforward because blur follows a slightly different approach than virtual background in the code.